templates/netlify/remix.init/index.js

```diff
@@ -0,0 +1,107 @@
+const inquirer = require("inquirer");
+const fs = require("fs/promises");
+const { join } = require("path");
+const PackageJson = require("@npmcli/package-json");
+
+const filesToCopy = [
+  ["README.md"],
+  ["netlify.toml"],
+  ["server.js"],
+  ["remix.config.js"],
+  ["vscode.json", join(".vscode", "settings.json")],
+  ["globals.d.ts"],
```

Comment:
I think we can just remove `globals.d.ts`, as everything that we're importing is already imported by the adapter in `globals.ts`

```suggestion
```